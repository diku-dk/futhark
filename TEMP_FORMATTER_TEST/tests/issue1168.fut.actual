9a42c5adb120442a5bb5def31cf09b6b
