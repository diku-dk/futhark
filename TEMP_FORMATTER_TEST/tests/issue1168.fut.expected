ec4d9b1788488f39b19783de3a9dd5b7
